scipy/special/_add_newdocs.py

```diff
@@ -8329,21 +8331,20 @@ def add_newdoc(name, doc):
 
     Negative binomial cumulative distribution function.
 
-    Returns the sum of the terms 0 through `k` of the negative binomial
+    Returns the sum of the terms 0 through `floor(k)` of the negative binomial
     distribution probability mass function,
 
     .. math::
-
-        F = \sum_{j=0}^k {{n + j - 1}\choose{j}} p^n (1 - p)^j.
+        F = \sum_{j=0}^{\lfloor k \rfloor} {{n + j - 1}\choose{j}} p^n (1 - p)^j
```

Comment:
You can split maths across multiple and this will avoid the need for noqa. An example of this is https://github.com/scipy/scipy/blob/0dfb5d600299af16b1482731def7c91f7662ec28/scipy/special/_add_newdocs.py#L1850-L1854
which renders fine https://scipy.github.io/devdocs/reference/generated/scipy.special.chndtr.html#scipy.special.chndtr